### PR TITLE
Fix: use dropout parameter

### PR DESCRIPTION
**Pull request by dhernandez0**
The dropout parameter passed to the class constructor is not used. The default p=0.5 of torch.nn.Dropout is always used instead.

### CODE REVIEW

**Review by smcdonagh — COMMENTED** [review]
OK
